roles/edpm_container_manage/tasks/delete_orphan.yml

```diff
@@ -20,6 +20,13 @@
   when:
     - edpm_container_manage_cli == 'podman'
 
+- name: Exclude Ceph containers from podman container list
```

Comment:
On one hand I'm wondering if this could be done as filter in the followup task. Otoh this is way more readable.